Simulator/simulation/simulator.js

```diff
@@ -2,17 +2,71 @@ const fs = require('fs');
 const House = require('./house');
 
 Simulator = {
```

Comment:
Sometimes class sometimes object? We should choose one and stick to it

Reply:
My thought was that we only want one instance of Simulator while we want several of house/windturbines thus the different uses

Reply:
🤷